site/en/blog/immutable-document-domain/index.md

```diff
@@ -112,7 +112,19 @@ To learn more about the security implications of setting `document.domain`, read
 ["Document.domain" page on
 MDN](https://developer.mozilla.org/docs/Web/API/Document/domain#setter).
 
-Chrome plans to make `document.domain` immutable from Chrome 101.
+Chrome plans to make `document.domain` immutable in the future.
+
+### How do I know my site is affected?
+
+If your website is affected by this change, Chrome will complain that `Setting
+document.domain will be deprecated.` in the DevTools console.
+
+It will also send a deprecation report. You can receive it if you have a server
```

Comment:
Maybe "Affected websites will also get sent with deprecation reports, if they have a reporting endpoint set up. Learn more about [using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api) with either existing report collection services or by building your own in-house solution."